docs/changes.rst

```diff
@@ -56,12 +50,6 @@ Released: not yet
   smaller (110 mb) but the same set of models and providers as the previous
   image.
 
```

Comment:
I notice that the "Known issues" header is missing.  Was that on purpose.  Should we leave it there just so we have a reminder to use in future versions even if we simply change the text to "No known issues"

Reply:
That is on purpose. The start of a new release adds it again, and the release removes it. We wanted to avoid maintaining known issues as a list, so we took the short way of just having a link. However, the set of open issues changes over time, and I think that does not go well with a change log. But I have no strong opinion on this.

Reply:
This release follows what we have done in all previous releases. I suggest to release it that way, and if we want to change anything, to do it for the next release.